### ISSUE TITLE
Latex elements get faded when exporting to PDF

### ISSUE THREAD

**Issue by tXJtqlb3bbeKD1yW8SV7** [zsviczian/obsidian-excalidraw-plugin#796]
When exporting to PDF (built-in exporter) the latex elements get faded.

Obs: I think the drawings get a little faded as well.

# Screenshots
![msedge_EgIHDBkspo](https://user-images.githubusercontent.com/88219689/188892650-e03b33c1-1b57-4124-8497-cd64d9abd041.png)

# Environment
- Plugin version: 1.7.19
- Obsidian version: v0.15.9
- Installer version: v0.15.9
- Operating system: Windows 10 Home Single Language 10.0.19044
- Legacy editor: off
- Base theme: dark
- Community theme: Minimal
- Snippets enabled: 0

# Things I've already tried

- Switching to light mode.
- Disabling the Minimal Theme.
- Beside using the own Excalidraw format, embedding the drawing as .svg and .png.

**Comment by zsviczian** [comment]
you get the faded images when you export to PNG as well?

**Comment by 02202** [comment]
Yep!

**Comment by 02202** [comment]
Yep, the PNG gets faded outside of obsidian too. I had tested it with the light mode on Excalidraw, but now, taking a look at how the image on obsidian light mode, it gets faded too.

# Obsidian Dark Mode (PNG embed, default theme):
![Obsidian_PSRvnquEbl](https://user-images.githubusercontent.com/88219689/189376233-0ae6479a-63c2-4c12-8d77-7890cf8466de.png)


# Obsidian Light Mode (PNG embed, default theme):
![Obsidian_daAVQLroz4](https://user-images.githubusercontent.com/88219689/189376339-c67fa3ab-31af-48f1-bf57-fbf30ce57af7.png)
